lib/stores/FileStore.js

```diff
@@ -168,6 +167,32 @@ class FileStore extends DataStore {
             });
         });
     }
+
+    binPath(file_id) {
+        return `${this.directory}/${file_id}`;
+    }
+
+    configPath(file_id) {
```

Comment:
I wonder if there is a better name than _config_, we don't need to beholden to it because the library was previous named prefixed with it.

Reply:
in the last commit I renamed it getFileInfo/writeFileInfo; think it is more descriptive and is similiar to tusd GetInfo/writeInfo